cmd/attack/redis.go

```diff
@@ -85,6 +86,23 @@ func NewRedisSentinelStopCommand(dep fx.Option, options *core.RedisCommand) *cob
 	return cmd
 }
 
+func NewRedisCacheLimitCommand(dep fx.Option, options *core.RedisCommand) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "cache-limit",
+		Short: "set maxmemory of Redis",
+		Run: func(*cobra.Command, []string) {
+			options.Action = core.RedisCacheLimitAction
+			utils.FxNewAppWithoutLog(dep, fx.Invoke(redisAttackF)).Run()
+		},
+	}
+
+	cmd.Flags().StringVarP(&options.Addr, "addr", "a", "", "")
+	cmd.Flags().StringVarP(&options.Password, "password", "p", "", "The password of server")
+	cmd.Flags().StringVarP(&options.CacheSize, "size", "s", "0", "The size of cache")
```

Comment:
what will happen if the cache size is 0? Do we need to check the value in validation?

Reply:
If user doesn't set `maxmemory` in redis config, it will be `0`, it is a default value. And redis will help us to validate this value, I don't think we need to do extra validation.

Reply:
OK